cli/batch/interfaces.py

```diff
@@ -0,0 +1,21 @@
+"""
+Provide implementation of the batch interfaces.
+"""
+
+
+class BatchInterface:
+    """
+    Implements batch interface.
+    """
+
+    def get_list(self, query=None):
+        """
+        Get a list of batches.
+
+        List of batches can be filtered by identifiers, start pagging, identifier of block's head, limit
```

Comment:
Remove `List of batches can..` from the interface. Leave in the implementation.

Reply:
Done.